components/NavBar/index.js

```diff
@@ -1,68 +1,242 @@
+        </div>
         <div
           className={
-            toggle ? styles.dropdownContent : styles.dropdownContentHide
+            mountedComponent
+              ? `${styles.navBarLogin}`
+              : `${styles.navBarLogin} d-none`
           }
         >
-          <a href="#">Link 1</a>
-          <a href="#">Link 2</a>
-          <a href="#">Link 3</a>
-          <a href="#">Link 4</a>
-          <a href="#">Link 5</a>
+          <Link href={LOGIN_URL}>
+            <a
+              className={
+                isLoggedIn ? `${styles.btnLogin} d-none` : `${styles.btnLogin}`
+              }
+            >
+              <button className={styles.btnLoginText}>
+                Sign In
+                <img
+                  className={styles.githubLogo}
+                  src={GITHUB_LOGO}
+                  alt="GitHub Icon"
+                  height="15px"
+                  width="15px"
+                />
+              </button>
+            </a>
+          </Link>
+          <div
+            className={
+              isLoggedIn ? `${styles.userGreet}` : `${styles.userGreet} d-none`
+            }
+          >
+            <Link href={PATHS.PROFILE}>
+              <a>
+                <div className={styles.userGreetMsg}>
+                  {isLoggedIn
+                    ? `Hello, ${userData.firstName}!`
+                    : `Hello, User!`}
+                </div>
+                <img
+                  className={styles.userProfilePic}
+                  src={
+                    isLoggedIn
+                      ? `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/${userData.userName}/img.png`
+                      : `${DEFAULT_AVATAR}`
+                  }
+                  alt="Profile Picture"
+                />
+              </a>
+            </Link>
+          </div>
         </div>
+        <ul
+          className={
+            toggle ? `${styles.navBarMenu} ${styles.active}` : styles.navBarMenu
+          }
+        >
+          <li className={styles.navBarLogoLi}>
+            <Link href={PATHS.HOME}>
+              <a>
+                <img
+                  src={RDS_LOGO}
+                  alt="home nav logo"
+                  height="50px"
+                  width="50px"
+                />
+              </a>
+            </Link>
+          </li>
+          <li className={styles.homeTab}>
+            <Link href={PATHS.HOME}>Home</Link>
+          </li>
+          <li>
+            <Link href={PATHS.WELCOME}>Welcome</Link>
+          </li>
+          <li>
+            <Link href={PATHS.EVENTS}>Events</Link>
+          </li>
+          <li>
+            <Link href={PATHS.MEMBERS}>Members</Link>
+          </li>
+          <li>
+            <Link href={PATHS.CRYPTO}>
+              <a className={styles.activeTab}>Crypto</a>
+            </Link>
+          </li>
+          <li>
+            <Link href={PATHS.STATUS}>Status</Link>
+          </li>
```

Comment:
Missing something?

Reply:
I don't think so. Can you please tell us what we have missed ??

Reply:
Reference: https://github.com/Real-Dev-Squad/website-my/pull/188#discussion_r777096269

Reply:
Ohh yes 💯

Reply:
Addressed in #246